README.rst

```diff
@@ -69,46 +66,82 @@ Every time you develop something in this repo
 
   # Open a PR and ask for review.
 
-License
--------
+Deploying
+=========
 
-The code in this repository is licensed under the AGPL 3.0 unless
-otherwise noted.
+TODO: How can a new user go about deploying this component? Is it just a few
+commands? Is there a larger how-to that should be linked here?
 
-Please see `LICENSE.txt <LICENSE.txt>`_ for details.
+PLACEHOLDER: For details on how to deploy this component, see the `deployment how-to`_
```

Comment:
should we add that the library works out of the box since maple?

Reply:
Yes, we can make it clear to say that it works out of the box for the lms and cms services since it is installed by default in edx-platform.
However we can start by saying that this component is a library and therefore is not deployed but instead is added as a requirement to IDAs.

We could also add an explanation about how one could add this to an IDA that is not the edx-plat.

Reply:
That works! Thanks